src/main/java/Mentos/task/Event.java

```diff
@@ -0,0 +1,21 @@
+package Mentos.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task{
+    protected LocalDateTime from;
+    protected LocalDateTime to;
```

Comment:
Since LocalDateTime is a noun, perhaps you can consider renaming these fields such that they sound more like nouns! (eg. fromDate or toDate)